redfish-client/redfish-client.py

```diff
@@ -314,23 +320,48 @@ def get_manager_info(manager_name, check_SSL):
     logger.info("Arguments parsed")
     logger.debug(arguments)
 
-    # Get $HOME environment.
+    # Get $HOME and $VIRTUAL_ENV environment variables.
     HOME = os.getenv('HOME')
+    VIRTUAL_ENV = os.getenv('VIRTUAL_ENV')
 
     if not HOME:
         print('$HOME environment variable not set, please check your system')
         logger.error('$HOME environment variable not set')
         sys.exit(1)
     logger.debug("Home directory : %s" % HOME)
+    
+    if VIRTUAL_ENV:
+        logger.debug("Virtual env : %s" % VIRTUAL_ENV)
+        
+    # Load master conf file
+    config = ConfigParser.ConfigParser(allow_no_value=True)
+    logger.debug("Read master configuration file")
+    master_conf_file_path = "/etc/redfish-client.conf"
```

Comment:
I really dislike the idea of running in place in fact. We need to find an agreement on that as indeed it has an important impact on some of the choices discussed in this PR.

Let's brainstorm in order to be able to satisfy all usages.